### ISSUE TITLE
Make the program available as a pdf download 

### ISSUE THREAD

**Issue by mattstratton** [devopsdays/devopsdays-web#8638]
Mentioned that having the ability to auto-generate a PDF version of the program would be great.

**Comment by toshywoshy** [comment]
I am working on a PDF schedule generated from the Pretalx Schedule using a webhook and LaTeX

**Comment by mattstratton** [comment]
I know we love pretalx, but I don't want to make it a dependency on a feature for the site :) 

That is to say, I like having that, i just don't know that having it would close this issue.

**Comment by mattstratton** [comment]
Yeah, I don’t think there is a way that we would directly be able to do it in Hugo, I was thinking that it might be using a separate thing maybe in a get of action that would generate them in the pipeline maybe?
